content/node/leanpocket.md

```diff
@@ -0,0 +1,94 @@
+---
+title: LeanPocket
+menuTitle: LeanPocket
+weight: 35
+description: Run multiple nodes on the same hardware without scaling your resource requirements.
+---
+
+LeanPocket (sometimes stylized as "LeanPOKT") is a configuration that allows multiple Pocket nodes running on a single server to consolidate their resources, resulting in better scalability and reduced hardware costs.
```

Comment:
You can already run multiple nodes on a single server. I believe the better wording is somewhere along the lines of "multiple Pocket nodes running on a single process"

Reply:
Great point, I'll make that change.